app/views/quizzes/proceed.coffee

```diff
@@ -79,7 +79,6 @@ displayNext = ->
   <% else %>
   renderNext('<%= j render partial: "quizzes/quiz_round",
                            locals: { hidden: true } %>')
-  $('[data-bs-toggle="popover"]').popover()
```

Comment:
I'm not shure, why you didn't insert an "initBootstrapPopovers()" here.
I tried a quiz with and without this additional line, but I didn't see that it changed anything in the behavior. So probably, this is fine as it is, but maybe double check this.

Reply:
forgot to add it here, fixed it (even though you didn't notice a change when deleting the line, but it doesn't harm to have it here as well)